system/spd.c

```diff
@@ -0,0 +1,978 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (C) 2004-2023 Sam Demeulemeester
+
+#include "display.h"
+
+#include "stdbool.h"
+#include "stdint.h"
+#include "string.h"
+
+#include "smbus.h"
+#include "spd.h"
+#include "jedec_id.h"
+
+// Functions Prototypes
+static void read_sku(char *sku, uint8_t slot_idx, uint16_t offset, uint8_t max_len);
```

Comment:
I think that this refactor is a good opportunity to reorder functions in this file and get rid of these early prototype declarations.
Sam, Martin ?

Reply:
Sure, done.